public/pages/Aliases/containers/DeleteAliasModal/DeleteAliasModal.tsx

```diff
@@ -44,14 +45,25 @@ export default function DeleteAliasModal(props: DeleteAliasModalProps) {
   const onConfirm = useCallback(async () => {
     if (services) {
       const result = await services.commonService.apiCaller({
-        endpoint: "indices.deleteAlias",
+        endpoint: "indices.updateAliases",
```

Comment:
Why not using delete alias API and specifying concrete index names to delete all of the aliases, maybe simpler than this?